ezugo-bio

```diff
@@ -0,0 +1,4 @@
++ # Github Ezugo bio
++
++ ## Details
++ **Favorite Color ** Red
```

Comment:
Red is :sparkles:!

What if you added your favorite vacation spot? :sunglasses: